ansible/roles/master/tasks/download_bins.yml

```diff
@@ -5,13 +5,9 @@
 - name: Create directory for current Kubernetes release
   file: path={{ kube_current_release_directory }} state=directory
 
-- name: Set a list of kube master binaries to download
-  set_fact:
-    kube_binaries: "{{ kube_master_binaries }}"
-
 - name: Extend the list of kube master binaries with kubelet
   set_fact:
-    kube_binaries: "{{ kube_master_binaries }} kubelet"
+    kube_master_binaries: "{{ kube_master_binaries + ['kubelet'] }}"
```

Comment:
This reason why this was not done in the introducing PR was the ``kube_master_binaries`` gets extended. And as each variable/fact has global scope, it could lead to unexpected behaviour if the ``kube_master_binaries`` gets used later after the ``download_bins``.